### PR TITLE
[OSL-2017] Update email address for our contact at Macsimum

### PR DESCRIPTION
**Pull request by sveinung**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2017] Add Bluth Company as a sponsor`*

### CODE REVIEW

**Comment by phrawzty** [comment]
The Travis build failed for reasons that are unclear to me. It appears unrelated to the changes you've proposed, and the Netlify preview is fine (including for the erroring event), so I'll go ahead and merge.

**Comment by sveinung** [comment]
Ok. Thanks :)

**Comment by phrawzty** [comment]
"Pull requests that have a failing status can’t be merged on a phone."

WELP nevermind then.

File: content/events/2017-oslo/sponsor.md [hunk 1]
```diff
@@ -31,4 +31,6 @@ If you have bought one of the sponsor packages we can also offer some add-ons.
 
 <img src="/events/2017-oslo/sponsorship-exclusive-packages.png" style="max-width: 100%; height: auto; width: auto\9;">
 
+For additional questions or requests please contact: vw@macsimum.no
```
Comment (bridgetkromhout) [inline content/events/2017-oslo/sponsor.md]: Sponsorship discussions should go through the organizer email list, which will allow the core team to help/guide if needed. Please don't direct such discussions elsewhere; thanks.

**Comment by sveinung** [comment]
Forget this one. I made a proper new one: https://github.com/devopsdays/devopsdays-web/pull/2516

**Pull request closed without merging**